components/brave_wallet/browser/asset_ratio_service.cc

```diff
@@ -94,6 +95,25 @@ std::vector<std::string> VectorToLowerCase(const std::vector<std::string>& v) {
   return v_lower;
 }
 
+constexpr char kEthereum[] = "ethereum";
+constexpr char kSolana[] = "solana";
+constexpr char kPolygon[] = "polygon";
+constexpr char kBitcoin[] = "bitcoin";
```

Comment:
Is it ok to have bitcoin here when wallet doesn't support it?

Reply:
It should be fine because Stripe does validation to make sure the wallet address supplied is consistent with the destination network, but I'll remove here since I don't see a reason for it to be there if we don't support Bitcoin.